adb_graphics/datahandler/gribdata.py

```diff
@@ -522,7 +525,7 @@ def corners(self) -> list:
         '''
 
         lat, lon = self.latlons()
-        if self.model == 'global':
+        if self.model in ['global', 'obs']:
```

Comment:
corner values are in a different order for global/obs vs the rest?

Reply:
Yes. They are stored and a 1-D array of lat/lon instead of meshed together in 2D arrays. Every grid box has the same lat along a lon, and vice versa.